crypto/keys.go

```diff
@@ -0,0 +1,25 @@
+package crypto
+
+import (
+	"crypto/ecdsa"
+	"log"
+
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/statechannels/go-nitro/types"
+)
+
+func GeneratePrivateKeyAndAddress() (types.Bytes, types.Address) {
```

Comment:
This might be overkill but 😆 
```suggestion
// GeneratePrivateKeyAndAddress generates a pseudo-random private key and it's corresponding address
func GeneratePrivateKeyAndAddress() (types.Bytes, types.Address) {
```